user_stories.md

```diff
@@ -0,0 +1,19 @@
+Marketing research team
+- As a market researcher I want to display three clickable images at a time to the user to observe which item image they choose of the three
+- As a market researcher I want to record data on the total number of clicks users have made on each item
+- As a market researcher I also want to see the percentage of times users clicked on an item when it was shown
+- As a market researcher I want to see how many times an item was displayed to calculate a proper analysis
+- As a market researcher I want the site to look clean and emulate other successful shopping sites
+
+Developer
+- As a developer I want to create an element (array) that will store all of my image elements
+- As a developer I want to create a functionality that will display three random images simultaneously
+- As a developer, I want to create a functionality that tracks each time an image is randomly iterated
+- As a developer I want to create a functionality that records an image selection upon a user clicking that particular image
+- As a developer I want to sum the aggregate amount of clicks per item
+- As a developer I also want to calculate a percentage of how many times an item was clicked versus how many times it was shown
+
+Focus Group
+- As a focus group member I want to be able to click on an image and for the page to respond to when I click
```

Comment:
This is better written as:

"As a focus group member, I want to click an image and be presented with a new grouping so that I can cast my votes on the options"

Reply:
Think about both cause and effect with all of your stories.